### PR TITLE
Add more information for DAA

### PR DESCRIPTION
**Pull request by SueChaplain**
- improve DAA section in the introduction
- list types of functions you can use
- list benefits
- link to API doc

Signed-off-by: Sue Chaplain <sue_chaplain@uk.ibm.com>

### CODE REVIEW

File: docs/introduction.md [hunk 1]
```diff
@@ -62,6 +62,20 @@ You can share class data between running VMs, which can reduce the startup time
 ### Native data operations
 If your Java application manipulates native data, consider writing your application to take advantage of methods in the Data Access Accelerator API.
 
+The following functions are provided:
+
+- Arithmetic, copying, and shifting operations for packed decimal data.
```
Comment (joransiu) [inline docs/introduction.md:67]: 'copying' should probably be 'comparison' -- though I noticed this text came directly from our API javadocs, which should be fixed too!   We might want to add validation operation to cover the checkPackedDecimal set of APIs.  I'd propose we go with:

> Arhithmetic, comparison, shifting and validation operations for packed decimal data.

File: docs/introduction.md [hunk 2]
```diff
@@ -62,6 +62,20 @@ You can share class data between running VMs, which can reduce the startup time
...
+- Routines to convert between decimal data types stored in byte arrays and Java binary types.
```
Comment (joransiu) [inline docs/introduction.md:68]: This bullet is a bit misleading; I read it as conversion between decimal types and Java binary types.  However, we also support conversion between different binary coded decimal types (i.e. Packed Decimal <-> External Decimal <-> Unicode Decimal).  

Perhaps we want to expand to say:
> Conversion operations between different binary coded decimal types and Java binary types.

File: docs/introduction.md [hunk 3]
```diff
@@ -62,6 +62,20 @@ You can share class data between running VMs, which can reduce the startup time
...
+- Marshalling operations: marshalling and unmarshalling Java binary types, such as `short`, `int`, `long`, `float`, and `double`, to and from byte arrays.
+
+You can gain a number of benefits by using the APIs provided:
+
+- Improved application performance by avoiding object creation and intermediate processing, which can also put pressure on the Java heap.
+- Hardware acceleration by automatically exploiting any known and available hardware features on specific platforms.
```
Comment (joransiu) [inline docs/introduction.md:74]: Maybe simply this statement by removing "any known and" 
> Hardware acceleration by automatically exploiting available hardware features on specific platforms.

File: docs/introduction.md [hunk 4]
```diff
@@ -62,6 +62,20 @@ You can share class data between running VMs, which can reduce the startup time
...
+- Platform independence for applications that are developed to take advantage of DAA.
```
Comment (joransiu) [inline docs/introduction.md:75]: Maybe we should expand DAA acronym here or add `(DAA)` above in the first sentence of this section.

**Comment by SueChaplain** [comment]
Thanks @joransiu - I made all the changes you suggested.

**Comment by pmhayward** [comment]
Jenkins doc stage

**Pull request merged**